eemeter/weather/cache.py

```diff
@@ -1,97 +1,81 @@
 import os
 import json
-import sqlite3
+from datetime import datetime
+from sqlalchemy import create_engine, Table, MetaData, Column, Integer, String, DateTime
+from sqlalchemy.sql import select, func
 
 
-class SqliteJSONStore(object):
+class SqlJSONStore(object):
 
-    def __init__(self, directory=None):
-
-        # creates the self.conn attribute
-        self._prepare_db(directory)
+    def __init__(self, url=None):
+        self._prepare_db(url)
 
     def __repr__(self):
-        return 'SqliteJSONStore("{}")'.format(self.directory)
-
-    def _get_directory(self):
-        """ Returns a directory to be used for caching.
-        """
-        directory = os.environ.get("EEMETER_WEATHER_CACHE_DIRECTORY",
-                                   os.path.expanduser('~/.eemeter/cache'))
-        if not os.path.exists(directory):
-            os.makedirs(directory)
-        return directory
+        return 'SqlJSONStore("{}")'.format(self.url)
 
-    def _prepare_db(self, directory=None):
+    def _get_url(self):
+        url = os.environ.get("EEMETER_WEATHER_CACHE_URL")
+        if url is None:
+            directory = "{}/.eemeter/cache".format(os.path.expanduser('~'))
+            if not os.path.exists(directory):
+                os.makedirs(directory)
+            url = "sqlite:///{}/weather_cache.db".format(directory)
```

Comment:
I did - reason I decided against it was I wanted the library to still be as portable as possible. Whether that was a good reason or not we can certainly discuss.